### PR TITLE
Add support for build results page

### PR DESCRIPTION
**Pull request by Manishearth**
Currently we get this by going to the buildbot grid, but if you're not on the latest five builds, you have to scroll through the waterfall, which is annoying.

Whipped this together as I was waiting for buildbot. Completely untested.

cc @jdm

---
This change is [Reviewable](https://reviewable.io/reviews/servo/homu/169)

### CODE REVIEW

**Comment by Manishearth** [comment]
image


it works!

I'm not sure how to write automated tests for this, AIUI we don't test the web interface at all.

**Comment by jdm** [comment]
Note that buildbot also supports http://build.servo.org/grid?width=20 (where 20 can be any number)

**Comment by Manishearth** [comment]
that is amazing

Though I do feel this still has utility, makes it easier to find (it's linked from the build queue), and we can also include it in the homu testfailed comment for more discoverability

File: homu/html/build_res.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+                <tr>
+                    <td class="hide">{{loop.index}}</td>
+                    <td>{{builder.name}}</td>
+                    <td class="{{builder.result}}"><a href="{{builder.url}}">{{builder.result}}</a></td>
+                </tr>
+                {% endfor %}
+            </tbody>
+        </table>
+
+        </script>
```
Comment (kennytm) [inline homu/html/build_res.html]: Is this a typo? There isn't an open `<script>` tag 🤔

**Pull request merged**